src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1253,7 +1256,14 @@ public static void applySslContext()
         {
             SSLFactory.validateSslContext("Internode messaging", conf.server_encryption_options, REQUIRED, true);
             SSLFactory.validateSslContext("Native transport", conf.client_encryption_options, conf.client_encryption_options.getClientAuth(), true);
+            // For JMX SSL the validation is pretty much the same as the Native transport
+            SSLFactory.validateSslContext("JMX transport", conf.jmx_encryption_options, conf.jmx_encryption_options.getClientAuth(), true);
             SSLFactory.initHotReloading(conf.server_encryption_options, conf.client_encryption_options, false);
+            /*
+            For JMX SSL, the hot reloading of the SSLContext is out of scope for CASSANDRA-18508.
```

Comment:
Can you please file a JIRA to handle this - I understand why it may not be doable at this time, and the issue pre-existed your change, but it would be good to be able to handle hot reloading and we should look at it soon.

Reply:
Done, https://issues.apache.org/jira/browse/CASSANDRA-20067